doc/classes/VisualShaderNodeCompare.xml

```diff
@@ -1,45 +1,62 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeCompare" inherits="VisualShaderNode" category="Core" version="3.2">
 	<brief_description>
+		A comparsion function for common types within visual shader graph.
 	</brief_description>
 	<description>
+		Compares [code]a[/code] and [code]b[/code] of [member type] by [member function]. Returns a boolean scalar. Translates to "if" instruction in shader code.
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="condition" type="int" setter="set_condition" getter="get_condition" enum="VisualShaderNodeCompare.Condition" default="0">
+			Extra condition which applied if [member type] setted to [constant CTYPE_VECTOR].
```

Comment:
Extra condition which _is_ applied if [member type] _is set_ to [constant CTYPE_VECTOR].